projects/storefrontlib/src/lib/cms-lib/navigation/navigation.service.ts

```diff
@@ -11,7 +11,11 @@ export class NavigationService {
    * @param root
    * @param itemsList
    */
-  public getNavigationEntryItems(nodeData: any, root: boolean, itemsList = []) {
+  public getNavigationEntryItems(
```

Comment:
Related to `createChildren()`:
* Please add `void`
* Please add types to parameters
* Can we add type to `children`